src/platform-includes/capture-error/apple.mdx

```diff
@@ -19,27 +19,37 @@ SentrySDK.capture(exception: exception)
 [SentrySDK captureException:exception];
 ```
 
-<PlatformSection supported={["apple.macos"]}>
+### Swift Errors
 
-## Capturing Uncaught Exceptions in macOS
+For Swift Errors conforming to the [Error Protocol](https://developer.apple.com/documentation/swift/error) the SDK
+sends the domain, code and the description of the Swift error.
+Before [sentry-cocoa 8.7.0](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#870) the SDK sends only
+the domain and error code.
```

Comment:
Is there a reason for these funny line breaks? It doesn't affect the output, but it's a little annoying

Reply:
No, not a reason. I removed them.